platform/linux-generic/include/odp_packet_io_internal.h

```diff
@@ -68,9 +68,13 @@ struct pktio_entry {
 	/* These two locks together lock the whole pktio device */
 	odp_ticketlock_t rxl;		/**< RX ticketlock */
 	odp_ticketlock_t txl;		/**< TX ticketlock */
-	uint8_t cls_enabled;            /**< classifier enabled */
-	uint8_t chksum_insert_ena;      /**< pktout checksum offload enabled */
 	uint16_t pktin_frame_offset;
+	struct {
```

Comment:
I wonder how to comment a commit comment in github, but here "Reserve memory" should probably be "save memory" or similar.

Reply:
Ok